### PR TITLE
Rename project files to the current application name

### LINKED ISSUE #4: Rename project assets to reflect name change
Rename the original "CSP" names in the project to reflect the new name "Langscaper."

Might be good to get this done and out of the way while the project is still in its infancy.

### PR DESCRIPTION
**Pull request by c-m-green**
Renames existing files to match the application name. This replaces the generic "CSP" name with something much more readily identifiable as part of the Langscaper project.

Note that you may have to reload the project in your IDE after these changes are applied, and you _might_ need to redo your run configuration(s). Otherwise, this _shouldn't_ break anything too badly...

(Also included: a somewhat cheeky update to the default Avalonia greeting.)

### CODE REVIEW

File: Langscaper/Langscaper.csproj [hunk 1]
```diff
@@ -9,21 +9,21 @@
...
-        <PackageReference Include="Avalonia.Fonts.Inter" Version="11.1.0"/>
+        <PackageReference Include="Avalonia" Version="11.1.0" />
+        <PackageReference Include="Avalonia.Desktop" Version="11.1.0" />
+        <PackageReference Include="Avalonia.Themes.Fluent" Version="11.1.0" />
+        <PackageReference Include="Avalonia.Fonts.Inter" Version="11.1.0" />
         <!--Condition below is needed to remove Avalonia.Diagnostics package from build output in Release configuration.-->
-        <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="11.1.0"/>
-        <PackageReference Include="CommunityToolkit.Mvvm" Version="8.2.1"/>
+        <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="11.1.0" />
+        <PackageReference Include="CommunityToolkit.Mvvm" Version="8.2.1" />
```
Comment (c-m-green) [inline Langscaper/Langscaper.csproj:23]: No idea why my editor thought to do this, but...

Comment (Catylist0) [reply Langscaper/Langscaper.csproj:23]: Yeah I was scratching my head a bit lol, it doesn't make a difference I suppose?

Comment (terahlunah) [reply Langscaper/Langscaper.csproj:23]: I shouldn't matter, let's hope others editors wont try to remove that space later 😅

**Comment by terahlunah** [comment]
LGTM

**Pull request merged**